MapboxCoreNavigationTests/DistanceFormatterTests.swift

```diff
@@ -66,7 +66,8 @@ class DistanceFormatterTests: XCTestCase {
         NavigationSettings.shared.distanceUnit = .mile
         distanceFormatter.numberFormatter.locale = Locale(identifier: "en-GB")
         
-        assertDistance(0,               displayed: "0 ft")
+        assertDistance(0,               displayed: "0 yd")
```

Comment:
0&nbsp;feet is more relatable than 0&nbsp;yards. I’m guessing 10&nbsp;yards is a good threshold under which feet is more appropriate.

Reply:
Do we really want to introduce yards at all? Currently we only use ft, right?

Reply:
Despite the distance to the sign is usually measured in meters, the sign itself, almost exclusively displays the distance in yards between 0-0.1 mile in the U.K. This seems to be the correct behavior.
ref:
https://en.wikipedia.org/wiki/Road_signs_in_the_United_Kingdom
https://www.gov.uk/government/uploads/system/uploads/attachment_data/file/203668/traffic-signs-manual-chapter-07.pdf#page=155
https://www.waze.com/forum/viewtopic.php?f=657&t=114107
https://wiki.waze.com/forum/viewtopic.php?f=3&t=29882